src/cli.ts

```diff
@@ -11,6 +12,8 @@ if (!globalThis.fetch) {
   (globalThis as any).Response = Response;
 }
 
+dotenv.config();
```

Comment:
env vars are already supported... is this just to support the .env file specifically as well?

Reply:
yep, in case people (like me) want to pick up vars from an env file. Maybe at some point we could also add a config file that gets inited to your home dir.